frontend/src/pages/PerpetratorTracker.js

```diff
@@ -27,7 +27,7 @@ export default function PerpetratorTracker(){
         let valuesEachDay = 0
 
         pains.forEach(list =>
-            list.map(painDto =>
+            list.forEach(painDto =>
```

Comment:
Think about refactoring this to make better use of javascrips array methods (I think reduce is what you want here). Due to the imperative style this function is kinda long and not so easy to understand. I think some helper function might improve this.